File: src/problems/longest-substring-without-repeating-characters/boilerplate-full/function.java
//USER_CODE_HERE
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String s = sc.next();
        Solution sol = new Solution();
        int result = sol.lengthOfLongestSubstring(s);
        System.out.println(result);
    }
}

File: src/problems/median-of-two-sorted-arrays/boilerplate-full/function.py
#USER_CODE_HERE
import sys
from typing import List

# Set recursion limit higher for recursive problems
sys.setrecursionlimit(2000)

if __name__ == "__main__":
    # Input reading for the specific problem
    nums1 = list(map(int, input().split())) # List of integers from space-separated input
    nums2 = list(map(int, input().split())) # List of integers from space-separated input

    sol = Solution()
    
    # Call the solution function
    result = sol.findMedianSortedArrays(nums1, nums2)
    
    # Print the result
    print(result)